src/components/statements/statements.ts

```diff
@@ -88,7 +89,16 @@ export async function viewStatement(ctx: IContext, params: IParameters): Promise
     orgGUIDs: [organization.metadata.guid],
   };
 
-  const events = await billingClient.getBillableEvents(filter);
+  let events;
+  try {
+    events = await billingClient.getBillableEvents(filter);
+  } catch {
+    return {
+      body: errorsTemplate.render({
+        errorMessage: 'Billing is currently unavailable, please try again later.',
+      }),
+    };
+  }
```

Comment:
I'm wondering if that's the best approach. This would require us to load the errors template in (not currently) all of the modules, and try and catch... Can we `throw new CustomErrorWhithAMessageThatItsDefinitionDoesntExistYet('Billing is currently unavailable, please try again later.')` and let the middleware handle it?